### DISCUSSION TITLE
Release nbgrader 0.6.0

### DISCUSSION THREAD

**Discussion by jhamrick** [jupyter/nbgrader#1644] [General]
I am planning to make a release for 0.6.0 after we get in the changes from the hackathon, so hopefully within the next few weeks!

If there is anything you'd really like to see in 0.6.0 that is currently not marked for 0.6.0, please let me know. (However, if it's a major change it's unlikely to be something that will make it in as I'd rather not keep pushing the release off).

**Comment by Sefriol** [comment]
It would be nice if this could be completed before the start of the semester. Any particular issue in this milestone that would require help? We might have some people over here at the university who could spend time on a few issues (or me ;) ).

Especially partial grading and feedback sharing are something what we are interested in, since those are currently done using some hacks on our part. Some of these changes are ofc in master already, but it would be nice to have a stable release of this asap.

Especially since most of the users probably won't use the current master for their production. With proper release, they might and then we can I iron out the raising issues much faster.

**Comment by kcranston** [comment]
I'd also like to see this get out before the start of the semester. I should be able to take a look at the needed testing for #1090 . I would really like to see the docs updated latest -> stable, because as a nbgrader newbie, I've been caught by a few things in `latest` that are already fixed in `stable`.
